commcare_connect/opportunity/views.py

```diff
@@ -685,11 +687,22 @@ def user_visits_list(request, org_slug=None, opp_id=None, pk=None):
     opportunity = get_opportunity_or_404(pk=opp_id, org_slug=org_slug)
     opportunity_access = get_object_or_404(OpportunityAccess, pk=pk, opportunity=opportunity)
     user_visits = opportunity_access.uservisit_set.order_by("visit_date")
-    user_visits_table = UserVisitTable(user_visits, org_slug=request.org.slug)
+    visit_filter = UserVisitFilter(request.GET, queryset=user_visits, managed_opportunity=opportunity.managed)
+    user_visits_table = UserVisitTable(
+        visit_filter.qs, org_slug=request.org.slug, template_name="django_tables2/bootstrap5.html"
```

Comment:
Why do we need to specify a template here (and didn't before), and can it be specified on the table model, rather than mid view

Reply:
This was added to enable pagination the default template `tabbed_tables.html` has HTMX related code for pagination that caused issues with tables loaded directly on pages.

Reply:
can it be set on the class? Or do we use the table elsewhere

Reply:
Added the template to the table class. [da4f59e](https://github.com/dimagi/commcare-connect/pull/475/commits/da4f59e3a2c6f5803dd4e5682ef0cbea3aa1d3d0)